test/date_types.test.js

```diff
@@ -95,7 +97,19 @@ describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with local TZ', funct
       },
     }, function(err, found) {
       assert.ok(!err);
-      assert.ok(found);
+      assert.equal(found.id, 1);
+      done();
+    });
+  });
+
+  it('should find model instance by date field passing string', function(done) {
```

Comment:
@darknos Test is failing locally for this. Why is this a valid test case to test for a `dateField` of type string when it expects a Date object?